Makefile

```diff
@@ -9,18 +9,32 @@ IMAGE_NAME := node-feature-discovery
 IMAGE_TAG_NAME := $(VERSION)
 IMAGE_REPO := $(IMAGE_REGISTRY)/$(IMAGE_NAME)
 IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
-
+K8S_NAMESPACE := kube-system
 
 all: image
 
-image:
+image: nfd-master.yaml nfd-worker-daemonset.yaml
 	$(IMAGE_BUILD_CMD) --build-arg NFD_VERSION=$(VERSION) \
 		-t $(IMAGE_TAG) ./
 
+%.yaml: %.yaml.template FORCE
+	@echo "$@: namespace: ${K8S_NAMESPACE}"
+	@echo "$@: image: ${IMAGE_TAG}"
+	@sed -E \
+	     -e s',^(\s*)name: node-feature-discovery,\1name: ${K8S_NAMESPACE},' \
```

Comment:
Could it be made even a bit more obvious what line in the template(s) this expression targets? E.g. having something like this in the template and the sed expression:
```
name: node-feature-discovery  # NFD namespace
```

Reply:
Thanks! This fixes the thing that bothered me a bit. :)